integration_tests/integration/recentArrivals/recentArrivals.cy.ts

```diff
@@ -3,13 +3,15 @@ import Page from '../../pages/page'
 import Role from '../../../server/authentication/role'
```

Comment:
there's a binary file [cypress/downloads/downloads.html](https://github.com/ministryofjustice/hmpps-welcome-people-into-prison-ui/pull/388/files#diff-707e98447df3abc4f9a958a39e41dbb76470f48fbaef89674200b6f06a0aadac). That presumably needs to be deleted and potentially git ignored?

Reply:
Not sure how that got in and I missed it when I checked the PR.
Deleted